### ISSUE TITLE
JAWS/Chrome "region" does not read root/parent element

### ISSUE THREAD

**Issue by shelbyelavsky** [fizzstudio/svg-aria-test#37]
This is an issue in reference to #25 which is related to the ["region" branch](https://github.com/fizzstudio/svg-aria-test/tree/aria_region).

<img width="1194" alt="Screenshot of JAWS in test environment. A red circle is highlighting text that reads 'A graphical display of data Each element displays different information' that is not spoken by or listed in the Speech History for JAWS." src="https://user-images.githubusercontent.com/98043076/153676970-973d238b-fc65-4bd2-bd3f-75bb904675a5.png">

### Issue Summary
JAWS does not read the SVG root/parent element (which should be the first grouping before continuing to the children element groups.) It goes into reading the first child group. This issue also occurs in the HTML example. 

### Video of issue
Example starts at 00:00:10
https://user-images.githubusercontent.com/98043076/153676474-5a880a6f-9308-4a4d-aeeb-2655271966e0.mp4

### Steps to reproduce
Use JAWS down arrow key once to navigate into the SVG space. (we would expect at least nine separate elements to navigate.)

### Expected behavior (Pass/Fail)
- **FAIL** We expect that a screen reader (SR) can navigate to each element with a role and aria-label using a keyboard (SVG, g, and all SVG primitives).
- **FAIL** We expect that the visual focus indicator moves to the correct element and has the correct bounding dimensions.
- **FAIL** We expect that the visual focus indicator will remain in view at all times and may even adjust the scroll position of the page while navigating
- **FAIL** The SVG is labelled by the chart's title and subtitle (these should be read as a single text when navigating to the root element).
- **FAIL** The title and subtitle are hidden from a screen reader because they are announced on their parent element instead. These elements should not be read out loud.
- **FAIL** The elements are grouped in pairs. The screen reader should be able to separately navigate to the group before navigating to each pair.
- **Pass** Each primitive (rect, circle, path, text) should be accessible by a screen reader and read the number and label for that element.
- **Pass** We expect that a SR will announce the aria-label correctly for each element, followed by its role.
- **Pass** The aria label should announce the number, followed by the type of primitive (eg "7 Rect"), even though the text may be displayed viusally in a different order. This is designed intentionally to ensure that the ARIA is working correctly.
- **FAIL** We expect that different SVG primitives (rect, circle, path, text) will be consistent with each other based on previous expectations.
- **FAIL** We expect that the SVG elements will provide the same experience as the HTML elements (below the SVG space).

### Technical Details
JAWS 2022.2112.24 ILM
Chrome Version 97.0.4692.99
Windows 11 Build 22000.438
